ext/opentelemetry-ext-system-metrics/tests/test_system_metrics.py

```diff
@@ -0,0 +1,212 @@
+            self.assertEqual(
+                observer.aggregators[(("type", "user"),)].current, 332277.48
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "nice"),)].current, 0.0
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "system"),)].current, 309836.43
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "idle"),)].current, 6724698.94
+            )
+
+    @mock.patch("psutil.virtual_memory")
+    def test_system_memory(self, mock_virtual_memory):
+        VirtualMemory = namedtuple(
+            "VirtualMemory",
+            [
+                "total",
+                "available",
+                "percent",
+                "used",
+                "free",
+                "active",
+                "inactive",
+                "wired",
+            ],
+        )
+        mock_virtual_memory.return_value = VirtualMemory(
+            total=17179869184,
+            available=5520928768,
+            percent=67.9,
+            used=10263990272,
+            free=266964992,
+            active=5282459648,
+            inactive=5148700672,
+            wired=4981530624,
+        )
+
+        meter = self.meter_provider.get_meter(__name__)
+        with mock.patch("opentelemetry.metrics.get_meter") as mock_get_meter:
+            mock_get_meter.return_value = meter
+            system_metrics = SystemMetrics(self.memory_metrics_exporter)
+            # pylint: disable=protected-access
+            observer = Observer(
+                None, "test-name", "test-desc", "test-unit", int, meter,
+            )
+            system_metrics._get_system_memory(observer)
+            self.assertEqual(
+                observer.aggregators[(("type", "total"),)].current, 17179869184
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "available"),)].current,
+                5520928768,
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "used"),)].current, 10263990272
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "free"),)].current, 266964992
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "active"),)].current, 5282459648
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "inactive"),)].current,
+                5148700672,
+            )
+            self.assertEqual(
+                observer.aggregators[(("type", "wired"),)].current, 4981530624
+            )
+            self.assertNotIn((("type", "percent"),), observer.aggregators)
+
+    @mock.patch("psutil.net_io_counters")
+    def test_network_bytes(self, mock_net_io_counters):
+        NetworkIO = namedtuple(
+            "NetworkIO",
+            ["bytes_sent", "bytes_recv", "packets_recv", "packets_sent"],
```

Comment:
Any reason to discard packets?

Reply:
Only reason I'm discarding them is that an observer needs a unit for the metrics it's observing, I picked to only observe bytes here.